### PR TITLE
added external link icon and removed price/dollar-sign

### LINKED ISSUE #7375: [PNI] Product page - header updates: remove pricing
- [x] Remove product price

Current page implementation:

image

- [x] Add external link icon. SVG icon here: https://drive.google.com/open?id=1Tb1mypj_5pfzhrZ53Hg6ONIkhIp4JxU2&authuser=sabrina%40mozillafoundation.org&usp=drive_fs 

Relevant part of the updated product page comp:

image

Comps on figma: https://www.figma.com/proto/GwdoUiY4fzVyeLfXsAJGPq/*Privacy-Not-Included-Website?page-id=548%3A72128&node-id=611%3A77813&viewport=283%2C48%2C0.08&scaling=min-zoom&starting-point-node-id=611%3A77813

### PR DESCRIPTION
**Pull request by danielfmiranda**
Closes #7375

**Link to sample test page**: https://foundation-s-7375-remov-p9munk.herokuapp.com/en/privacynotincluded/various-road-challenge/

**Steps to test**: 

1. Visit the link above
2. Verify that the external link icon looks correct
3. if so, testing is complete! 

## Checklist

**Changes in Models:**
- [x] [Are my changes backward-compatible](https://github.com/mozilla/foundation.mozilla.org/blob/main/docs/workflow.md#django-migrations-what-to-do-when-working-on-backward-incompatible-migrations). If not, did I schedule a deploy with the rest of the team?

### CODE REVIEW

**Review by sabrinang — CHANGES_REQUESTED** [review]
- the right side looks a bit thin/cropped on firefox (looks fine on chrome though)
image

**Comment by danielfmiranda** [comment]
Hi @sabrinang! Thanks for the catch, I have updated the icon width to prevent it from being cutoff and have re-requested review.

File: source/sass/buyers-guide/views/product.scss [hunk 1]
```diff
@@ -134,6 +134,19 @@
...
+  .company-external-link {
+    display: inline-flex;
+    align-items: center;
+
+    &::after {
+      content: " ";
+      width: 17px;
+      height: 16px;
+      background: url(../_images/buyers-guide/external-link.svg) no-repeat;
+      margin-left: 8px;
```
Comment (Pomax) [inline source/sass/buyers-guide/views/product.scss:146]: this feels very close to just `width: 1em`, `height: 1em` and `margin-left: 0.5em`, which probably makes more sense if we're setting the icon based on the text it goes with. @fessehaye, thoughts?

**Review by Pomax — CHANGES_REQUESTED** [review]

File: source/sass/buyers-guide/views/product.scss (hunk 1 above)
Comment (danielfmiranda) [reply source/sass/buyers-guide/views/product.scss]: Hi @Pomax! Awesome catch, I agree that the original sizing was very close to 1em and 0.5em. 
I have updated the css to use the EM values now, and can also confirm that they solve the icon cutoff issue from above as well.

**Review by sabrinang — APPROVED** [review]
The icon appears good now 👍 thanks Daniel!

**Review by Pomax — APPROVED** [review]

**Pull request merged**